src/components/ProfileCard.tsx

```diff
@@ -0,0 +1,67 @@
+import {User} from "next-auth";
+import {api} from "~/utils/api";
+import {timeAgo} from "~/utils/time";
+import {ProfilePicture} from "./ProfilePicture";
+import {Spinner} from "./Spinner";
+
+interface ProfileCardProps {
+    dateJoined?: Date,
+    name: string
+    imgUrl?: string | null,
+    userId: string,
+}
+
+export const ProfileCard: React.FC<ProfileCardProps> = ({
+    dateJoined,
+    name,
+    imgUrl,
+    userId,
+}) => {
+    const { data: nBlogPosts } = api.blogPost.count.useQuery({
+        userId: userId 
```

Comment:
This query is not invalidated when creating/deleting a blog post

You can test this by:
1. go to the profile page
2. create a new blog post
3. Observe that the '# of comments' indicator is not changed

Reply:
fixed